### PR TITLE
DOC-1426 : moved command docker-compose to docker compose

### PR DESCRIPTION
**Pull request by HarshEviden**
- Moved from docker-compose to docker compose

### CODE REVIEW

**Review by suityou01 — DISMISSED** [review]
mr-bean-thumbs-up

**Review by suityou01 — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -20,11 +20,11 @@
   "license": "OGL-UK-3.0",
   "dependencies": {
     "applicationinsights": "2.3.5",
-    "ffc-messaging": "2.7.0",
+    "ffc-messaging": "^2.10.1",
     "joi": "17.11.0",
     "log-timestamp": "0.3.0",
     "moment": "2.29.4",
-    "mssql": "10.0.1",
+    "mssql": "^11.0.1",
```
Comment (samplackett) [inline package.json]: Make these packages exact versions - so 11.0.1 rather than ^11.0.1

File: package.json [hunk 2]
```diff
@@ -20,11 +20,11 @@
   "license": "OGL-UK-3.0",
   "dependencies": {
     "applicationinsights": "2.3.5",
-    "ffc-messaging": "2.7.0",
+    "ffc-messaging": "^2.10.1",
```
Comment (GodsonLeigh) [inline package.json]: ^ remove this so the version number is set 2.10.1 is what we would use

File: package.json (hunk 1 above)
Comment (GodsonLeigh) [inline package.json]: Same as above, just ^ to remove

**Review by GodsonLeigh — APPROVED** [review]
small-approve

**Pull request merged**